Makefile

```diff
@@ -16,6 +16,9 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+# Include additional files
+include ./make/*.mk
```

Comment:
I can't really remember why now, but there was some reason we added this at the bottom of the Makefile https://github.com/Kuadrant/kuadrant-operator/blob/main/Makefile#L392

Reply:
Seems like it somehow connected to the MAKEFILE_LISTS variable - the one that Make creates just before execution to hold the list of all the files :thinking:  [This](https://stackoverflow.com/a/64394417/8716469) looks like the problem that you might've had along with the solution. Would you have a strong preference on having `include` statements in the end of the file?